app/Model/User.php

```diff
@@ -935,4 +935,45 @@ public function getMembersCount() {
 		}
 		return $usersPerOrg;
 	}
+
+	public function findAdminsResponsibleForUser($id){
+		$userOrg = $this->find('first', array(
+			'conditions' => array(
+				'User.id' => $id,
+			),
+			'contain' => array(
+				'Organisation' => array('fields' => array('id')),
```

Comment:
You don't actually need Organisation here, you only want the org_id, which is contained in the User model itself (it's a foreign key)